docs/docs/reference/other-new-features/parameter-untupling.md

```diff
@@ -38,6 +38,26 @@ Generally, a function value with `n > 1` parameters is converted to a
 pattern-matching closure using `case` if the expected type is a unary
 function type of the form `((T_1, ..., T_n)) => U`.
 
+More specifically, the adaptation is applied to the mismatching formal
+parameter list. In particular, the adaptation is not a conversion
+between function types. That is why the following is not accepted:
+
+```scala
+val combine: (Int, Int) => Int = _ + _
+xs.map(combine)
+```
+
+An explicit conversion for this case may be provided in user code, as usual:
+
+```scala
+import scala.language.implicitConversions
+inline transparent implicit def `enable untupling`(f: (Int, Int) => Int): ((Int, Int)) => Int =
+  p => f(p._1, p._2)
+xs.map(combine)
```

Comment:
How about `xs.map(Function.tupled(combine))` instead? That seems more usual than a transparent inline implicit conversion...

Reply:
This was to address "why isn't my function value adapted?" and show that the expected conversion works (and is equivalent or as efficient as expected with transparent inline). I accidentally switched the order of modifiers after seeing them printed that way in printed tree output. Ha.

Reply:
That makes sense and I agree that the inline definition should be more efficient, but I think I would still be more comfortable merging a more "straightforward" solution, it seems like it would be a better fit for `parameter-untupling.md`!

Reply:
updated the description, keeping the conversion but on the down low.